qiskit_addon_dice_solver/dice_solver.py

```diff
@@ -56,7 +56,7 @@ def solve_hci(
     nelec: tuple[int, int],
     ci_strs: tuple[Sequence[int], Sequence[int]] | None = None,
     spin_sq: float = 0.0,
-    select_cutoff: float = 5e-4,
+    select_cutoff: float = 2147483647,
```

Comment:
`solve_hci` is a general function and should be left alone. Instead, just change the value in line 244 where this function is called by `solve_fermion`.